CharlieBackend.Business/Services/ScheduleServiceFolder/Helpers/SchedulesEventsDbEntityVerifier.cs

```diff
@@ -99,5 +100,27 @@ public async Task<string> ValidateScheduledEventId(long id)
 
             return result;
         }
+
+        public async Task<string> ValidateCreateScheduleEventRequestAsync(CreateSingleEventDto request)
+        {
+            StringBuilder error = new StringBuilder(string.Empty);
```

Comment:
is it necessary to pass empty string to StringBuilder constructor?

Reply:
We did the same in previous validators. It isn't needed, thanks.